proto/delta_sharing/v1/delta_sharing.proto

```diff
@@ -174,6 +174,110 @@ message GetTableVersionResponse {
   int64 version = 1;
 }
 
+message GetTableMetadataRequest {
+  // The share name to query. It's case-insensitive.
+  string share = 1 [(buf.validate.field).string.min_len = 1];
```

Comment:
just FYI - the `buf.validate...` annotations are neither required nor used right now :). But since there are constraints on these fields, and I wanted to test the `protovalidate` framework, I added some and just kept them, since they also do no harm.